### ISSUE TITLE
Snapshot of database

### ISSUE THREAD

**Issue by nikyraja** [rcpch/rcpch-audit-engine#1099]
Please can we have a snapshot of the database that the analyst can connect to in STATA to use for the annual analyses?

Could this be taken on 15 January 2025 at 12:00am?

**Comment by mbarton** [comment]
To confirm how we'd do this, we would clone the `e12_live` database entirely into a new database which can then be accessed.

https://stackoverflow.com/questions/876522/creating-a-copy-of-a-database-in-postgresql

Rather than disconnect clients as that Stack Overflow answer suggests we can simply spin down the E12 app (set replicas to 0), take the snapshot and then spin it back up again.

@nikyraja this will be a manual process, is it OK to do it early the day after or late the day before? To avoid staying up until midnight :)

**Comment by nikyraja** [comment]
Yes of course! Could we aim to do this at 9/10am on 15 January 2025 please?

**Comment by mbarton** [comment]
Confirming this will now take place 9/10AM Monday 20th of Jan as we extended the deadline

**Comment by nikyraja** [comment]
Yes please @mbarton

**Comment by mbarton** [comment]
I have done this already today, the snapshot database is called `e12_live_snapshot_250120_0633`
